### ISSUE TITLE
Small KaTeX Rendering Bug (fails to render in certain situations)

### ISSUE THREAD

**Issue by rdipietro** [notable/notable#447]
- **OS Version**: MacOS High Sierra
- **Notable Version**: 1.3.0

### Current behavior

```
# KaTeX Testing

- $m$, an array of memory (say $m = \{m_i\}$) with each $m_i$ being a vector
- An array of memory (say $m = \{m_i\}$) with each $m_i$ being a vector
```

This renders as follows:

![screen shot 2019-02-12 at 11 07 23 am](https://user-images.githubusercontent.com/5150559/52649668-b52c0f80-2eb6-11e9-81e3-7260eeca6c68.png)

### Expected behavior

The rendering of `$m = m_i$` should not depend on what came earlier, near the beginning of the bullet.

**Comment by RomanKornev** [comment]
Looks like #427 is causing this again.

**Comment by RomanKornev** [comment]
Replacing `$m$` with `$m0$` fixes it, so it seems to be caused by single-symbol KaTeX expressions.

**Comment by fabiospampinato** [comment]
Fixed in v1.6.
